src/wp-includes/blocks.php

```diff
@@ -1056,9 +1056,60 @@ function apply_block_hooks_to_content( $content, $context, $callback = 'insert_h
 		$after_block_visitor  = make_after_block_visitor( $hooked_blocks, $context, $callback );
 	}
 
-	$blocks = parse_blocks( $content );
+	$block_allows_multiple_instances = array();
+	// Remove hooked blocks from the list if they have `multiple` set to false and are already
+	// present in `$content`.
```

Comment:
[Multi-line comments are supposed to be done like this:](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-inline-comments)

```suggestion
	/*
	 * Remove hooked blocks from the list if they have `multiple` set to false and are already
	 * present in `$content`.
	 */
```

Reply:
> This comment would be more clear if it mentioned the list they're removed from:
>
>> Remove hooked blocks from $hooked_block_types if…

a3c1b823f1